guide/inheritance-wallet/onboarding-cosigners.md

```diff
@@ -0,0 +1,149 @@
+---
+layout: guide
+title: Onboarding cosigners
+description: This page describes how Alice sets up the wallet in her app and activates it on her signing device.
+nav_order: 3
+parent: Inheritance wallet
+permalink: /guide/inheritance-wallet/onboarding-cosigners/
+main_classes: -no-top-padding
+image_base: /assets/images/guide/inheritance-wallet/onboarding-cosigners/
+images_bob-export-config:
+    - file: bob-export-config/onboard-alice-home
+      alt: Home screen showing three tasks that are associated with the wallet for the user to complete.
+      caption: A contextual task list helps users to complete all the necessary steps to safely use their new wallet.
+    - file: bob-export-config/onboard-alice-dialog
+      alt: Sharing dialog showing a QR code and sharing options.
+      caption: Bob exports the wallet configuration, so that Alice can set up the wallet on her phone.
+    - file: bob-export-config/onboard-alice-mark-complete
+      alt: Home screen showing three tasks that are associated with the wallet for the user to complete.
+      caption: After Alice has successfully set up the wallet on her end, Bob checks off the task. 
+    - file: bob-export-config/onboard-alice-completion-dialog
+      alt: Screen showing a confirmation dialog.
+      caption: Bob confirms that all cosigners have been onboarded. In our use case, Bob and Alice are the only cosigners.
+    - file: bob-export-config/onboard-alice-done
+      alt: Home screen showing two tasks associated with the wallet.
+      caption: The cosigner onboarding task is gone from the list and Bob can continue with the the next task.
+images_alice-import-config:
+    - file: alice-import-config/app-home-initial
+      alt: Screen showing an empty app home screen.
+      caption: Alice taps the plus button to start adding the wallet. 
+    - file: alice-import-config/add-wallet-sheet
```

Comment:
It feels a bit weird that Alice is "Recovering" a wallet when Bob is trying to "Onboard a co-signer". Could Alice have an option to "Onboard as co-signer", or could that maybe be explained in the description ("Use this feature also if you're onboarding as a co-signer...")?

Reply:
Good point. I think there are two possibilities:
1. Change the wording to "Import wallet", which doesn't clash so much for that use case.
2. Add a fourth option called "Join wallet". This would technically be the same feature as "Recover", since both would just import a wallet configuration. But it addresses the user intent more explicitly and presents options which matches what they are probably looking for (I want to join or recover a wallet) and since the app now knows what the user wants to do, it can tailor the flow that follows to that context.

I'd go for option 2, personally. What do you think?

Reply:
After more thought, I chose to stick with three options (Create wallet, import wallet, and provide key) and just update the wording.